src/main/java/com/bgsoftware/superiorskyblock/core/menu/button/impl/menu/BiomeButton.java

```diff
@@ -88,15 +92,13 @@ public void onButtonClick(SuperiorSkyblockPlugin plugin, MenuBiomes superiorMenu
             return;
         }
 
-        File file = new File(plugin.getDataFolder(), "menus/biomes.yml");
-        YamlConfiguration cfg = YamlConfiguration.loadConfiguration(file);
-
         if(cfg.isConfigurationSection("biome-environment")) {
             ConfigurationSection configurationSection = cfg.getConfigurationSection("biome-environment");
-            String environment = configurationSection.getString(event.getResult().name().toUpperCase(Locale.ENGLISH), "Normal");
+            String result = event.getResult().name().toUpperCase(Locale.ENGLISH);
+            String environment = configurationSection.getString(result, "NORMAL");
 
-            if(clickedPlayer.getWorld() != null && clickedPlayer.getWorld().getEnvironment() != World.Environment.valueOf(environment.toUpperCase())) {
-                Message.CANT_CHANGE_BIOME.send(clickedPlayer, event.getResult().name().toLowerCase(Locale.ENGLISH));
+            if(clickedPlayer.getWorld() != null && clickedPlayer.getWorld().getEnvironment() != World.Environment.valueOf(environment)) {
```

Comment:
In case environment is not set in config upper-case, Environment#valueOf will throw an error.